GVRf/Framework/backend_daydream/build.gradle

```diff
@@ -6,15 +6,42 @@ repositories {
     flatDir {
         dirs 'src/main/libs'
     }
+    // For GVR components, the official source of the .aars is the Google
+    // repository.
+    maven {
+        url "http://google.bintray.com/googlevr"
+    }
 }
 
 android {
-    compileSdkVersion 24
+    compileSdkVersion 21
     buildToolsVersion '24.0.3'
 
     defaultConfig {
         minSdkVersion 19
-        targetSdkVersion 24
+        targetSdkVersion 21
+        ndk {
+            File curDir = file('./')
+            curDir = file(curDir.absolutePath)
+            String libsDir = curDir.absolutePath +
+                    '/build/intermediates/exploded-aar/Framework/framework/unspecified/jni/armeabi-v7a/'
+
+            moduleName = "gvrf-daydream"
+            abiFilter "armeabi-v7a"
+            cFlags "-fexceptions -std=c++11 " + " -I" + curDir.absolutePath + "/." +
+                    "./framework/src/main/jni/ "
+            cFlags += "-I" + curDir.absolutePath + "/src/main/jni"
+            cFlags += "-I" + curDir.absolutePath + '/../framework/src/main/jni/util '
+            cFlags += "-I" + curDir.absolutePath + '/../framework/src/main/jni/contrib '
+
+            stl "gnustl_shared"
+            ldLibs curDir.absolutePath + "/src/main/jniLibs/armeabi-v7a/libgvr.so"

```

Comment:
Missed the *.so because of the gitignore. 

Added it to the PR:

https://github.com/Samsung/GearVRf/pull/988/commits/56214ab7a09008c061ed2a4fb81f8db1a487cd99